pkg/oci/client/client.go

```diff
@@ -103,6 +104,16 @@ func New(config *Config) (Interface, error) {
 		subnetCache: cache.NewTTLStore(subnetCacheKeyFn, time.Duration(24)*time.Hour),
 	}
 
+	compartmentOCID := config.Auth.CompartmentOCID
```

Comment:
Assignment here seems to be redundant. 

```go
if config.Auth.CompartmentOCID == "" {
```

Reply:
👍 good catch. This was copy/pasta from the original PR. i'll update